tools/decompress_baserom.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: © 2024 ZeldaRET
+# SPDX-License-Identifier: MIT
```

Comment:
Why does this file get a copyright text and license identifier? None of the other source files have it.

(ditto for z64compress_wrapper.py)

Reply:
We have been adopting this on our scripts to make their license explicit. We have some tools in the repo that we don't even know where they came from and it have been the topic of discussion for some time.
As a reference we have been adopting the license identifier for scripts on other repos like MM and AF

Reply:
iirc the cc0 licensing effort explicitly states that tools can and should detail its own license if it wants to. I also am not sure on the details though, @Dragorn421 would know

Reply:
Not sure what to explain but it's good that tools are explicitly licensed

The cc0 agreement is a blanket cover for everything so we (and repo users in general) can do literally what we want without consulting all contributors (caveat: not all contributors agreed)

You may argue that since anghelo agreed as roughly "all my contributions are cc0" you may still take the MIT code he contributes as cc0, but that's not the spirit of it. Plus this MIT licensing is much more specific than the blanket cc0 statement so feels like it prevails
In the end it's the honor system anyway, what matters if you do have honor is to feel like you're abiding by the authors' wishes (which is easy to check when the wishes are expressed with licenses)

Reply:
btw if we want to use a different license for those scripts I'm happy to change it.
I used MIT since that's the one I use as a default for almost everything, and I'm happy to gave up those scripts contributions to ZeldaRET instead of explicitly having my name on it